hypershift-operator/controllers/manifests/controlplanepkioperator/reconcile.go

```diff
@@ -0,0 +1,80 @@
+package controlplanepkioperator
```

Comment:
This file (and the test) don't follow the convention in this controller, but the convention is to put everything in disjoint places and many thousand-line-files so I hope that's ok